.travis.yml

```diff
@@ -16,6 +16,8 @@ branches:
 before_script:
   - pip install -U pip setuptools wheel typing
   - pip install cwltool cwltest html5lib
+  - pip install -r requirements.txt
 
 script:
   - make RUNNER=cwltool unittest-examples
+  - make html
```

Comment:
Is this needed? Read the docs/netlify should be doing the building for us...

Reply:
Ah, good point, it's indeed building it again over here. Let me remove that. Thanks!